### DISCUSSION TITLE
Sponsor button shows up on the header bar, but the "Sponsor this project" sidebar section is empty

### DISCUSSION THREAD

**Discussion by ymc9** [community/community#84721] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I just finished setting up sponsor for my github org, created a FUNDING.yml file in my org's .github repo. After turning on sponsor button for my repo, it shows up on the header bar:

<img width="1233" alt="image" src="https://github.com/community/community/assets/104139426/dd932c7d-f20b-4a8e-8c22-81e7e6ef45bb">

However on the sidebar section it has empty content:

<img width="563" alt="image" src="https://github.com/community/community/assets/104139426/024b1c19-c700-47f2-b36f-93a25ac9aab5">

How should I get the sponsor link shown up there? Thanks!

Repo: https://github.com/zenstackhq/zenstack
Org: https://github.com/zenstackhq

**Comment by hejoefs** [comment]
Hey Yiming,

Could you check your FUNDING.yml configuration and ensure the file in your organization's .github repository contains the correct information and formatting for the sponsor link? Verify the repository where you've enabled the sponsor button is correctly linked to the organization's sponsorship settings. Sometimes, the repository settings might override the organization settings.

Please try to think about contacting directly and submitting a ticket on [Github Support](https://support.github.com/request) if the problem continues.

**Reply by ymc9** [reply]
Thanks! I didn't know I could put an org id inside FUNDING.yml. Changing it from my userid to org id fixed the issue.

**Reply by hejoefs** [reply, marked as answer]
My pleasure! I'm glad you fixed it, wish you the best! :)
